frontend/src/components/Tile/Tile.stories.tsx

```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react'
+import { BiLockAlt, BiMailSend } from 'react-icons/bi'
+import { Stack, UnorderedList } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+import _ from 'lodash'
```

Comment:
nit: import specific function instead of the whole library?

Reply:
sure!